src/main/java/nextstep/courses/domain/Student.java

```diff
@@ -0,0 +1,59 @@
+package nextstep.courses.domain;
+
+import java.util.Objects;
+
+public class Student {
+    private final String studentId;
+    private SelectStatus selectedStatus = SelectStatus.UNSELECTED;
+
+    public static Student SelectedStudent(String studentId) {
+        return new Student(studentId,SelectStatus.SELECTED);
+    }
+
+    public static Student UnSelectedStudent(String studentId) {
+        return new Student(studentId);
+    }
+
+    private Student(String studentId, SelectStatus selectedStatus) {
+        this.studentId = studentId;
+        this.selectedStatus = selectedStatus;
+    }
+
+
+    private Student(String studentId) {
+        this.studentId = studentId;
+    }
+
+    public boolean isSelected() {
+        if(selectedStatus == SelectStatus.SELECTED) {
+            return true;
+        }
+        return false;
```

Comment:
다음과 같이 줄여서 작성해 볼 수도 있을 것 같습니다.
```suggestion
        return selectedStatus == SelectStatus.SELECTED;
```